spaceship.zsh

```diff
@@ -293,11 +293,18 @@ _is_git() {
   command git rev-parse --is-inside-work-tree &>/dev/null
 }
 
-# Check if the current direcotory is in a Mercurial repository
+# Check if the current directory is in a Mercurial repository.
 # USAGE:
-#   _is_hg
+#   if [ $(_is_hg) ]
 _is_hg() {
```

Comment:
Curious to know whether this improves the load time. For me, their performances are almost identical. How does this performs on large repos ?